### PR TITLE
Update Desktop dash; Clarify KPI explore; Add YoY Measures

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: KPI/views/firefox_desktop_usage_2021.view.lkml [hunk 1]
```diff
@@ -136,6 +136,7 @@ derived_table: {
   }
 
   measure: new_profiles_cumulative_2021 {
+    hidden: yes
```
Comment (fbertsch) [inline KPI/views/firefox_desktop_usage_2021.view.lkml:139]: I ended up hiding a bunch of fields we specifically need for the topline dashboard numbers. I don't expect people would want to query these in the explore.

File: KPI/views/firefox_desktop_usage_2021.view.lkml [hunk 2]
```diff
@@ -167,55 +171,64 @@ derived_table: {
...
 
   measure: delta_from_target_new_profiles {
+    hidden: yes
     type: number
     value_format: "0.000%"
     sql: (${recent_new_profiles_cumulative} / (${prediction.recent_cum_new_profiles_target}) ) - 1 ;;
   }
 
   measure: delta_from_forecast_new_profiles_count {
+    label: "New Profiles: Absolute Delta from Forecast"
```
Comment (fbertsch) [inline KPI/views/firefox_desktop_usage_2021.view.lkml:216]: I also clarified a bunch of measures using `label`. I think the explore now looks cleaner and maybe a bit less intimidating.

File: KPI/views/firefox_desktop_usage_2021.view.lkml [hunk 3]
```diff
@@ -254,4 +268,57 @@ derived_table: {
    ;;
   }
 
+  # YoY Measures
```
Comment (fbertsch) [inline KPI/views/firefox_desktop_usage_2021.view.lkml:271]: Though these come from a join, I expose them in this table just for simplicity.

File: KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml [hunk 4]
```diff
@@ -450,8 +449,13 @@
     series_types: {}
     title_hidden: true
     listen:
+      Channel: firefox_desktop_usage_2021.channel
+      Activity Segment: firefox_desktop_usage_2021.activity_segment
+      OS: firefox_desktop_usage_2021.os
+      Attributed (Yes / No): firefox_desktop_usage_2021.attributed
+      Country Name: firefox_desktop_usage_2021.country_name
```
Comment (fbertsch) [inline KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml:456]: This fixes the issue Klukas noticed, where topline numbers weren't updated when filters changed

File: KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml [hunk 5]
```diff
@@ -375,7 +374,7 @@
               From Target New Profiles Count}], showLabels: true, showValues: true,
         unpinAxis: false, tickDensity: default, tickDensityCustom: 5, type: linear}]
     reference_lines: [{reference_type: line, range_start: max, range_end: min, margin_top: deviation,
-        margin_value: mean, margin_bottom: deviation, label_position: left, color: "#1A73E8",
+        margin_value: mean, margin_bottom: deviation, label_position: center, color: "#000000",
```
Comment (fbertsch) [inline KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml:377]: All the lines are now black and the labels are in the center (for both `At Forecast` and `Moving Avg`). I did this to make it more obvious what was/wasn't a reference line.

File: KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml [hunk 6]
```diff
@@ -538,10 +547,21 @@
...
-    width: 15
+    width: 24
     height: 9
+  - name: " (4)"
+    type: text
+    title_text: ''
+    subtitle_text: ''
+    body_text: "*Running slow? If no one has queried a slice before, it will take\
+      \ a 2-3 minutes to build the forecasting models. From then on, queries against\
+      \ that slice should run within 10-15 seconds.*"
```
Comment (fbertsch) [inline KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml:560]: This is new text at the top of the dash.
Screen Shot 2021-03-26 at 2 41 00 PM

**Review by irrationalagent — APPROVED** [review]

**Pull request merged**